editor/plugins/script_editor_plugin.h

```diff
@@ -116,7 +117,7 @@ class ScriptEditorBase : public Control {
 };
 
 typedef SyntaxHighlighter *(*CreateSyntaxHighlighterFunc)();
-typedef ScriptEditorBase *(*CreateScriptEditorFunc)(const Ref<Script> &p_script);
+typedef ScriptEditorBase *(*CreateScriptEditorFunc)(const RES &p_script);
```

Comment:
the same here. I would make it really obvious for ppl who work on it, that any resource "could be" edited. so calling the property `p_resource` makes sense to me?